src/vs/base/browser/ui/menu/menu.ts

```diff
@@ -321,7 +321,7 @@ class SubmenuActionItem extends MenuActionItem {
 	}
 
 	private createSubmenu() {
-		if (!this.parentData.submenu) {
+		if (!this.parentData.submenu && this.builder) {
```

Comment:
@sbatten how can this happen that we react on click but have not been created?

Reply:
@bpasero converting to runoncescheduler fixed the issue, reverted this change